packages/@n8n/benchmark/scenarios/js-code-node/js-code-node.script.js

```diff
@@ -5,6 +5,13 @@ const apiBaseUrl = __ENV.API_BASE_URL;
 
 export default function () {
 	const res = http.post(`${apiBaseUrl}/webhook/code-node-benchmark`, {});
+
+	if (res?.status !== 200) {
```

Comment:
From what I can tell, response is always defined - it's the response body that can be `undefined`. Same for others.

```ts
/**
 * Refined response.
 * Exposes body with best possible type.
 */
export interface RefinedResponse<RT extends ResponseType | undefined> extends Response {
    body: RefinedResponseBody<RT>;
}
```

Reply:
Alright, I'll remove the question marks